extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/SimpleScheduler.java

```diff
@@ -188,13 +185,8 @@ SimpleTrigger createTrigger(String invokerClass, CronParser parser, Scheduled sc
         }
     }
 
-    // Keep it public so that we can reuse the logic in the quartz extension
```

Comment:
Maybe we should extract these two static methods to a dedicated class? I'm not sure about the name though...

Reply:
Following the same approach in other extensions, I've created a `SchedulerUtils` and moved these two methods and also some static methods that were part of the interface `SchedulerContext` and were also related to configuration.

I left these methods as static but we could make them instance methods.

Let me know if it looks better now.

Reply:
The current version looks perfect! Thanks.